agent.py

```diff
@@ -247,10 +247,9 @@ def doDisease(self):
         diseaseCount = len(self.diseases)
         if diseaseCount == 0:
             return
-        neighborCells = self.cell.neighbors.values()
         neighbors = []
-        for neighborCell in neighborCells:
-            neighbor = neighborCell.agent
+        for cell in self.cell.neighbors.values():
```

Comment:
On second thought, this change should be reverted. It doesn't gain us anything and incurs an extra function call without added clarity for what's happening.